Dockerfile

```diff
@@ -1,18 +1,82 @@
-FROM continuumio/miniconda3
+FROM nvidia/cuda:11.2.0-cudnn8-devel-ubuntu20.04 AS build
 
-WORKDIR /app
+# RNCAN certificate; uncomment (with right .cer name) if you are building behind a FW
+# COPY NRCan-RootCA.cer /usr/local/share/ca-certificates/cert.crt
+# RUN chmod 644 /usr/local/share/ca-certificates/cert.crt && update-ca-certificates
 
-# Create the environment:
-COPY environment.yml .
-RUN conda env create -f environment.yml
+ARG PYTHON_VERSION=3.9
+ARG CONDA_PYTHON_VERSION=3
+ARG CONDA_DIR=/opt/conda
 
-# Make RUN commands use the new environment:
-SHELL ["conda", "run", "-n", "geo_deep_env", "/bin/bash", "-c"]
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends git wget unzip bzip2 build-essential sudo \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
-# Make sure the environment is activated:
-RUN echo "Make sure flask is installed:"
-RUN python -c "import flask"
+# Install miniconda
+ENV PATH $CONDA_DIR/bin:$PATH
+RUN wget --quiet https://repo.continuum.io/miniconda/Miniconda$CONDA_PYTHON_VERSION-latest-Linux-x86_64.sh -O /tmp/miniconda.sh && \
+    echo 'export PATH=$CONDA_DIR/bin:$PATH' > /etc/profile.d/conda.sh && \
+    /bin/bash /tmp/miniconda.sh -b -p $CONDA_DIR && \
+    rm -rf /tmp/* && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
-# The code to run when container is started:
-#COPY run.py .
-ENTRYPOINT ["conda", "run", "--no-capture-output", "-n", "geo_deep_env", "python"]
\ No newline at end of file
+## Variables 
+ARG GIT_HASH=f7e6033
```

Comment:
This hash refers to an old state of gdl. latest version on develop is:
version: v2.0.1
hash: 23abf68

Reply:
`GIT_HASH` is a build arg.  We can edit the Dockerfile with a new `GIT_HAS` periodically (e.g. when tagging a new version) but in the general case we'll use whatever commit hash at build time e.g. for CI.

```Dockerfile
docker build --build-arg GIT_HASH=23abf68 ...
```